src/Profile/DeviceConfig.cpp

```diff
@@ -100,29 +100,21 @@ LoadPath(const ProfileMap &map, DeviceConfig &config, unsigned n)
 {
   char buffer[64];
   MakeDeviceSettingName(buffer, "Port", n, "Path");
-  return map.Get(buffer, config.path);
-}
-
-static bool
-LoadPortIndex(const ProfileMap &map, DeviceConfig &config, unsigned n)
-{
-  char buffer[64];
-  MakeDeviceSettingName(buffer, "Port", n, "Index");
-
-  unsigned index;
-  if (!map.Get(buffer, index))
+#ifdef _WIN32 
+  // on Windows don't use the form 'COM1:' anymore, use 'COM1'
+  if (map.Get(buffer, config.path)) {
+    if (config.path[config.path.length() - 1] == ':') {
+      config.path[config.path.length() - 1] = 0;
+      // map.Set(buffer, config.path);  // unfortunately map is a const object!
```

Comment:
.... what????

Reply:
An overwriting of this profile setting isn't necessary - so I will remove this line. If the user make a new setting this don't have any colon!

Reply:
I still have no idea what that means or what your code means or what that code comment means.